src/App.js

```diff
@@ -1,5 +1,136 @@
+import { Console } from "@woowacourse/mission-utils";
+import InputView from "./InputView.js";
+import Validation from "./validation.js";
+import OutputView from "./OutputView.js";
+import { MENU } from "./constants/menu.js";
+import { EVENT, EVENT_DAYS, OFFER_MENU } from "./constants/event.js";
+
 class App {
-  async run() {}
+  /** 방문 예정 날짜 @type {number} */
+  #date;
+  /** 주문 메뉴 @type {Map<string, string>} */
+  #menusMap;
+
+  #totalAmount = 0;
+  #discount = { christmas: 0, weekdays: 0, weekends: 0, special: false, isOffer: false };
+  #totalDiscountAmount = 0;
+  #finalAmount = 0;
+
+  async run() {
+    OutputView.printWelcome();
+    this.#date = await this.#getDate();
```

Comment:
제가 잘몰라서 질문을 드립니다!
App.js를 보니 모든 return값들을 전역 private 필드에 넣으셔서 그거에 대한 질문입니다!
```suggestion
    const date= await this.#getDate();
```
이렇게 받지않으시고 #date (전역 private 변수)로 설정해서 받으시는 이유가 있으신가요?? 궁금합니다!

Reply:
이 부분은 저두 계속 고민중이라서 답변을 못드릴 것 같아요ㅠㅠ
이럴 때 private 필드를 사용하는게 맞는지 말씀하신대로 const로 선언해서 사용하는게 맞는건지...
아직 잘 모르겠습니다 ㅠㅠ

this.#date가 자주 사용되고 클래스 내에서 전역적으로 사용된다고 생각해서 필드 선언으로 하긴 했습니다..!
다만, 이렇게 하니 메서드들이 너무 의존성이 높아져서 리팩토링하면서 메서드들을 함수로 분리하고 있습니다! 파라미터로 받아서 return값을 내보내도록이요. 이렇게 하고나니 굳이 필드로 선언하지 않는게 더 유리해 보이긴 합니다...허허 혼란 투성이 ㅠ